traffic_ops/traffic_ops_golang/topology/topologies.go

```diff
@@ -150,6 +150,17 @@ func (topology *TOTopology) Validate() error {
 	rules["empty cachegroups"] = topology_validation.CheckForEmptyCacheGroups(topology.ReqInfo.Tx, cacheGroupIds, dsCDNs, false, nil)
 	rules["required capabilities"] = topology.validateDSRequiredCapabilities()
 
+	//Get current Topology-CG for the requested change.
+	topoCachegroupNames := topology.getCachegroupNames()
+	userErr, sysErr, _ = dbhelpers.CheckTopologyOrgServerCGInDSCG(topology.ReqInfo.Tx.Tx, dsCDNs, topology.Name, topoCachegroupNames)
+	if userErr != nil {
+		return fmt.Errorf("%s", userErr.Error())
+	}
+	if sysErr != nil {
+		fmt.Println("System Error", sysErr)
```

Comment:
This should use `log.Errorf` instead and should include a more precise context (e.g. `log.Errorf("validating topology: %s", sysErr.Error())`

Reply:
fmt.println was supposed to be cleaned up. That is my mistake. And i'll add a better log line.